lib/cli-engine.js

```diff
@@ -93,13 +95,21 @@ function calculateStatsPerFile(messages) {
     return messages.reduce((stat, message) => {
         if (message.fatal || message.severity === 2) {
             stat.errorCount++;
+            if (message.fix) {
+                stat.fixableErrorCount++;
+            }
         } else {
             stat.warningCount++;
+            if (message.fix) {
+                stat.fixableWarningCount++;
+            }
         }
         return stat;
     }, {
         errorCount: 0,
-        warningCount: 0
+        warningCount: 0,
+        fixableErrorCount: 0,
+        fixableWarningCount: 0
```

Comment:
If I'm not mistaken, it's possible for formatters to compute this on their own without needing a change to core. Is there a reason to add this to the report object?